src/wmnds/patterns/cookies/cookies-manager/_cookies-manager.njk

```diff
@@ -0,0 +1,53 @@
+{% macro wmndsCookiesManager(params) %}
+
+{# Imports of components #}
+{% from "wmnds/components/form-elements/checkboxes/_checkboxes.njk" import wmndsCheckboxes %}
+{% from "wmnds/components/button/_button.njk" import wmnButton as primary %}
+
+<div class="wmnds-p-md wmnds-cookies-manager">
+  <div class="wmnds-container wmnds-col-1 wmnds-col-sm-3-4 wmnds-col-lg-3-5">
+    <form class="wmnds-grid wmnds-cookies-manager__form" action="#">
```

Comment:
I'm not sure that the form needs the `.wmds-grid` class (and by extension the elements within don't need `.wmnds-col-1`)